### DISCUSSION TITLE
Synchronize with Apple Reminder sort

### DISCUSSION THREAD

**Discussion by Nadias-jp** [mpdifran/Memento-Issues#37] [Ideas]
Currently, the Sort of the list and task are not synchronized. Please support it.

**Comment by mpdifran** [comment]
Hey, thanks for logging this issue! Unfortunately Apple doesn't expose task ordering to third party devs. 

I spent a bunch of time when I started building Memento trying to get it to work to no avail!

**Reply by Nadias-jp** [reply]
It seems that Reminder can be "Sort" in iOS 14.5, will that have a positive effect on Memento?
https://www.iphonelife.com/content/how-to-sort-reminders-your-iphone

**Reply by mpdifran** [reply]
Yeah I noticed that! My guess is no, but I'll double check.

**Comment by mpdifran** [comment]
Added this task to investigate: https://github.com/mpdifran/Memento-Issues/issues/40
